examples/set-namespace/README.md

```diff
@@ -0,0 +1,26 @@
+# Set Namespace
+
+The `set-namespace` KRM config function adds or replaces the
+`metadata.namespace` field on all resources except for [those known to be
+cluster-scoped]. This example invokes the set-namespace function using
+declarative configuration.
+
+## Function invocation
+
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/set-namespace .
+kpt fn run set-namespace/configs --fn-path set-namespace/functions
```

Comment:
why do you need `fn-path`?

Reply:
I think it's cleaner not to set the namespace on the function config, only on the configs in the package dir set-namespace/configs. So I put the function config in a separate fn-path set-namespace/functions.

Reply:
Underlying issue is that function configs should not be processed by functions. That seems like a bug and may be related to https://github.com/GoogleContainerTools/kpt/issues/630

Reply:
Same as for annotate-config example. Let's merge these now and update them when there's consensus.